util/token.go

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) 2020-2021 The Decred developers
```

Comment:
See previous comment about regex sanitizing tokens.

Reply:
Done.  Tokens are now validated using the this regexp:
https://github.com/lukebp/politeia/blob/tlog/util/token.go#L26